### ISSUE TITLE
`Scopes`: The pagination shows Incorrect data

### ISSUE THREAD

**Issue by duttarnab** [GluuFederation/gluu-admin-ui#230]
The pagination shows 1 - 20 of 20. But there total 67 rows. It should show 1 - 20 of 67.
The user should be able to navigate to the next page by clicking on the > button (which is disabled even though there are 67 items).

**Comment by syntrydy** [comment]
I don't think this is an issue. The pagination is based on data available in the redux store, not the data available in the database.
To avoid unnecessary load, the **limit** param is used to reduce the amount of data to fetch from the database.

**Comment by duttarnab** [comment]
Hi @nynymike, could you please provide your opinion on this ticket.
